### PR TITLE
23w51b yet more stuff

### PR DESCRIPTION
**Pull request by apple502j**
Conflict to be resolved later

### CODE REVIEW

**Comment by apple502j** [comment]
Conflict resolved, @Shnupbups please make sure to SCREAM.

File: mappings/net/minecraft/entity/EntityAttachments.mapping [hunk 1]
```diff
@@ -1,9 +1,8 @@
 CLASS net/minecraft/class_9066 net/minecraft/entity/EntityAttachments
-	FIELD field_47751 DEFAULT Lnet/minecraft/class_9066;
```
Comment (apple502j) [inline mappings/net/minecraft/entity/EntityAttachments.mapping:2]: I couldn't find this field in the class. What's this? A ghost? 👻

Comment (Shnupbups) [reply mappings/net/minecraft/entity/EntityAttachments.mapping:2]: Hmm, maybe it was removed in the b snapshot? Most of my mapping was done on a

File: mappings/net/minecraft/entity/ai/brain/task/BreezeUtil.mapping [hunk 2]
```diff
@@ -0,0 +1,8 @@
+CLASS net/minecraft/class_9075 net/minecraft/entity/ai/brain/task/BreezeUtil
```
Comment (Shnupbups) [inline mappings/net/minecraft/entity/ai/brain/task/BreezeUtil.mapping]: if you prefer this name, please remove the `BreezeMovementUtil.mapping` in the same package

**Review by Shnupbups — CHANGES_REQUESTED** [review]
Whoops, sorry about all the forgetting to use SCREAMING_SNAKE_CASE, I must have been really tired haha. I think with the `babyDimensions` ones specifically I just started with the `AbstractDonkeyEntity` one, which isn't `static final` for some reason and thus camelCase is fine, and assumed others would be the same without looking closely. As for the `dateTimeFormatter` one... that was just extra sloppy on my part haha.

File: mappings/net/minecraft/entity/EntityAttachments.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/EntityAttachments.mapping:2]: Yeah I think it was removed in the b snapshot, and only existed in a.

**Review by Shnupbups — APPROVED** [review]

**Pull request merged**